src/components/validationStatus/errorListUtil.js

```diff
@@ -0,0 +1,13 @@
+import uniqueId from 'lodash/uniqueId';
+
+export default function errorList(validationErrors) {
+  return Object.entries(validationErrors).flatMap(([errorKey, errors]) => {
+    return errors.flatMap(error => {
+      return {
+        ...error,
+        errorKey,
+        ...{ 'errorId': `${ uniqueId('id') }_${ error.id }` },
```

Comment:
I don't think this needs to be descructured like this. Can it be set up as a key/value on the returned object?
```js
return {
  ...error,
  errorKey,
  errorId: `${ uniqueId('id') }_${ error.id }`
}
```

However, `uniqueId` is simply returning an auto-incrementing value, and at that point you may as well use the index as the key in the `v-for` in the template. I think what would work better as a unique key is the error ID (which I guess is the node ID in our case) and the error message.

The way it's set up now with `uniqueId`, if there is an actual bug and the exact same error gets duplicated in the list, no error will be thrown since it'll be assign a unique ID anyway.

The ID and message (node ID and error message) should be unique, and if it's not, we should expect to see a duplicate key warning.

Reply:
Yeah. I wasn't sure if I should try to turn the message into an id.